src/web.rs

```diff
@@ -46,6 +47,18 @@ pub struct WebImpl {
     width: u32,
 }
 
+pub enum Canvas {
```

Comment:
Would be nice to document this enum. Also, does it need to be `pub`?

Reply:
Done.